Sources/Valkey/Commands/ServerCommands.swift

```diff
@@ -478,25 +472,19 @@ public enum COMMANDLOG {
     @_documentation(visibility: internal)
     public struct LEN: ValkeyCommand {
         public enum _Type: RESPRenderable, Sendable, Hashable {
```

Comment:
we will need to add code gen tricks so that this isn't an API break anymore. Or we just support 6.2 and above with https://github.com/swiftlang/swift-evolution/blob/main/proposals/0487-extensible-enums.md